src/scales/modifiedLogScale.ts

```diff
@@ -98,8 +98,15 @@ module Plottable.Scales {
       super._setDomain(transformedDomain);
     }
 
```

Comment:
So, how exactly is this different than the previous behavior? As far as I can tell, modifiedLogScale is still returns _untransformedDomain, which was indicated as the source of the bug. I can see from your fiddle (thank you for that!) that the modified log scale is rendering, but what about this change set fixes it?